metron-analytics/metron-profiler-client/pom.xml

```diff
@@ -323,27 +167,20 @@
                                   </excludes>
                                 </filter>
                               </filters>
-                            <transformers>
-                               <transformer
-                                  implementation="org.apache.maven.plugins.shade.resource.DontIncludeResourceTransformer">
-                                  <resources>
-                                    <resource>.yaml</resource>
-                                    <resource>LICENSE.txt</resource>
-                                    <resource>ASL2.0</resource>
-                                    <resource>NOTICE.txt</resource>
-                                  </resources>
-                                </transformer>
-                                <!-- UNCOMMENT THIS IF YOU NEED TO REGENERATE THE BEST GUESS NOTICES FILE WHICH REQUIRES PRUNING EVERY RELEASE -->
```

Comment:
Is this no longer valid?

Reply:
No, did not need to change.  I can restore.  Just got included from one of my failed attempts to fix this.